workflow/cases/atm_free_forecast.yaml

```diff
@@ -0,0 +1,56 @@
+case:
+  places:
+    workflow_file: layout/free_forecast_gfs.yaml
+    #BASE_CPLIC: /scratch2/NCEPDEV/climate/climpara/S2S/IC
+
+  settings: 
+    SDATE: 2013-04-01t00:00:00
+    EDATE: 2013-04-01t00:00:00
+
+    cplflx: .false.
+    print_esmf: .true.
+    nems_temp: 'med_atm_ocn_ice'
+    nems_temp_cold: 'medcold_atm_ocn_ice'
+    mom6ic_prepared: .false.
+    KEEPDATA: NO
+    ic_source: cfsr
+
+  nsst:
+    NST_MODEL: 0
+
+  output_settings:
+    OCN_INTERVAL: 24
+    FHOUT_GFS: 6
+    FHMIN_GFS: 0
+    FHMAX_GFS: 48
+    FHMAX_HF_GFS: 0
+    FHOUT_HF_GFS: -1
+
+  fv3_gfs_settings:
+    CASE: C384
+    LEVS: 65
+    DELTIM: 450
+    SEEDLET: 10
+    CCPP_SUITE: FV3_GFS_v15p2_coupled
```

Comment:
If running uncoupled, might it be better to set the suite as "FV3_GFS_v15p2"? That will add NSST and remove sfc_cice. And then associated namelist changes for NSST would have to be made where appropriate, right?
Debatable on whether to include "FV3_GFS_v15p2" in the list of CCPP suites to build, since that adds to build time and very few people will be using coupled workflow to run uncoupled (at least for a while)

Reply:
You can of course change the CCPP suite as long as it is currently supported. But then you've added another degree of freedom into your experiment (CFSR/GFS IC, and CCPP_SUITE)

Reply:
You will have to make sure you compile the model with "FV3_GFS_v15p2" when you install the model.  This is not the default behavior and would have to be manually changed. I do think you will need to change this to run coupled versus uncoupled.  As this was copied directly from the coupled mode, you will have to change any variables here to be different if you want (such as turning on NSST, which is set to 0 as of now).

Reply:
Okay, I'm fine with it as-is. Better not to add degrees of freedom and fall down a rabbit hole at this point.